core/src/replay_stage.rs

```diff
@@ -361,6 +363,7 @@ impl ReplayStage {
                 Err(e)?;
             }
         }
+        Self::update_confidence_cache(bank_forks, tower, lockouts);
```

Comment:
I think the most expensive part of this function will the the `aggregate_stake_lockouts`. Do we have a bench for this when the number of banks gets large, should we offload this work to a separate thread?

Reply:
Ah, separate thread is a great idea. I'll work that up.